TIL/daily/2025-03-17.md

```
@@ -0,0 +1,119 @@
+
+메소드를 경유한 필드 조작입니다. 외부에서 직접 멤버 변수에 직접 접근하는 것을 제한하고, 내부에서만 접근 및 수정 가능하도록 제어할 수 있습니다.
+
+- getter: 읽기 전용 프로퍼티를 구현할 때 사용
+- setter: 쓰기 전용 프로퍼티를 구현할 때 사용
+
+```dart
+class Hero {
+		String name;
+		int _hp; //private 지정
+
+		int get hp => _hp
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)